### PR TITLE
feat(ced): add building with containers workshop for CED 2022

### PR DESCRIPTION
**Pull request by nimbinatus**

### CODE REVIEW

File: building-with-containers/README.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+* An [Azure account](https://www.pulumi.com/registry/packages/azure-native/installation-configuration/#credentials?utm_source=da&utm_medium=referral&utm_campaign=workshops&utm_content=ced-fall2022-workshops). The free tier should be all we need, but any costs accrued will be minimal if any appear at all.
+* [Docker](https://docs.docker.com/get-docker/) set up on your machine to build the container.
+
+### About this workshop
+
+In this workshop, we will explore secrets, stacks, inputs and outputs, and more.
+
+## Lab 1 - Setup
+
+Let's set up our sandbox application using a [Pulumi template for containers on Azure with Go](). First, make a new directory, and change into it.
```
Comment (aureq) [inline building-with-containers/README.md:34]: There may be a missing link here.

File: building-with-containers/azure-go/answers/lab4-hints.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Hints
+
+- Where have you hunted for the secret?
```
Comment (aureq) [inline building-with-containers/azure-go/answers/lab4-hints.md:3]: Could we replace 'hunted' with 'searched' as a non-violent word?

**Review by aureq — CHANGES_REQUESTED** [review]
2 minor suggestions but otherwise, looks good to me.

File: building-with-containers/azure-go/answers/lab4-hints.md (hunk 2 above)
Comment (nimbinatus) [reply building-with-containers/azure-go/answers/lab4-hints.md:3]: Good call!

File: building-with-containers/README.md (hunk 1 above)
Comment (nimbinatus) [reply building-with-containers/README.md:34]: Ha, caught. Yep; fixing now. Thanks!

Comment (nimbinatus) [reply building-with-containers/README.md:34]: Note: The updated link in the incoming commit will be correct once https://github.com/pulumi/docs/pull/8182 actually lands.

**Review by aureq — APPROVED** [review]

**Pull request merged**